File: src/test/java/com/roger/student/StudentTest.java
package com.roger.student;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class StudentTest {

    @Test
    void highest() {
        Student student = new Student("Roger", 60, 80);
        Assertions.assertEquals(80, student.highest());
    }

    @Test
    void getAverage() {
        Student student = new Student("Roger", 60, 80);
        Assertions.assertEquals((60+80)/2, student.getAverage());
    }
}

File: src/test/kotlin/com/roger/kotlin/StudentTest.kt
package com.roger.kotlin

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

import org.junit.jupiter.api.Assertions.*

internal class StudentTest {

    @Test
    fun grading() {
        val student = Student("Roger", 60, 80)
        Assertions.assertEquals('C', student.grading())
    }

    @Test
    fun getAverage() {
        val student = Student("Roger", 60, 80)
        Assertions.assertEquals((60+80)/2, student.getAverage())
    }

    @Test
    fun highest() {
        val student = Student("Roger", 60, 80)
        Assertions.assertEquals(80, student.highest())
    }
}